### ISSUE TITLE
Review and evaluate accessibility artifact request

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#47400]
## User Story
I am a a VFS team member trying to prepare accessibility artifacts for Staging Review but am having difficulty providing testing artifacts because I do not understand what I need to provide for review which makes me feel unprepared and confused.

I am an accessibility specialist on the Governance team trying to complete a Staging Review but I am unsure how thoroughly it's already been reviewed for accessibility because it's unclear who has completed reviews prior to Staging and what kind of accessibility knowledge they have which makes me feel like I might be duplicating effort.

Assignee: @briandeconinck 
Peer Reviewer: @allison0034 

## Description
In the Staging Review discovery, the below comments were provided with regards to accessibility:
- Foundational and Advanced testing don't have artifact templates associated with them.
- In general, Advanced testing isn't super actionable / doesn't translate to a sharable artifact.
- When they do fill in the a11y artifacts section, it's not actually that useful to me!
- Nowhere for VFS teams to indicate if they've worked with an a11y prior to review

Goals:
 - Understand what accessibility testing the VFS team has completed.
 - Find a way to connect known issues from VFS team to our Staging Review findings

### Extended description
Here's a description of the problem posted in #accessibility-ops when soliciting feedback, which goes into a little more detail:

> Background: VFS teams are required to complete Foundational accessibility testing prior to staging, and are encouraged to complete Advanced testing if they're able to.
> Foundational testing consists of:
> axe scans, including using axe in e2e tests
> WCAG color contrast requirements
> content zoom (200%, 300%, 400%)
> keyboard navigation
> Advanced testing is a lot less defined, and is basically "test with a screen reader and spot check with WAVE."
> Additionally, when a team prepares for staging, they're asked to provide these artifacts related to accessibility:
> Completed TestRail Platform accessibility test plan, OR
> Document in your product folder that indicates all required checks from the how to prepare for an accessibility staging review doc are complete
> List of known issues with links to tickets (when applicable)
> 
> In terms of what we actually get for those artifacts... it's mixed
> 
> Some teams don't provide anything. Some teams do the TestRail part but don't provide much detail about anything besides checking off boxes. A handful of teams have developed their own document template for completing foundational testing
> 
> But the key takeaways seem to be:
> 1. Many teams may not know what they're supposed to be submitting.
> 2. When I'm conducting a review, I'm not actually getting much useful information from what they are submitting and it's not making my reviews go any faster. (Even if the box is checked in TestRail that says they did content zoom testing, I'm still going to test that myself too.)
> 
> I'll also add: From the artifacts that do get submitted, I usually don't know if a team has had an a11y person involved at all or what kind of experience the person doing testing might have.
> (Example, without naming names: A team that had known axe failures but misunderstood the message axe was giving them and incorrectly dismissed it as a false positive.)
> 
> All of that is to say, I think we can do better!
> So here are some open questions for everyone in the group. I'd appreciate feedback from other accessibility specialists, but I'd love feedback from folks in this channel that don't have "accessibility" in their job title, especially if you're on a VFS team.
> 
> Is it worth asking for artifact(s) documenting pre-staging accessibility testing?
> What kind of information would be reasonable to ask teams to provide to document their testing? (Or, what can I reasonably ask for that's more than just a checkbox saying a step is completed?)
> What additional resources would help teams complete pre-staging testing?
> If a team is working with Shared Support or with another a11y specialist -- or has sufficient a11y experience that they're doing really robust testing on their own -- how can we avoid me duplicating effort by re-doing tests they've already done?
> 
> And I'm very open to thoughts and suggestions beyond those questions! This is all in the open-ended discovery phase :slightly_smiling_face:

### Quick stats
Looking at the [last 12 staging reviews](https://docs.google.com/spreadsheets/d/1KLNPpPQpNh5whtW_1SrJNi6C6cMBo1qQzGwlxpQ6xjY/edit#gid=0) (excluding DST components):
- 2 of 12 used the TestRail a11y testing template
- 3 of 12 provided no a11y artifacts whatsoever
- 2 of 12 provided a list of known a11y issues but no documentation of what testing took place (doesn't mean there was none, just means we don't know)
- 1 of 12 provided a copy of the axe scans for each page of their product, but no other testing documentation
- 5 of 12 provided their own markdown checklist for tracking Foundational and/or Foundational + Advanced testing.

## Impacted Artifacts
None. End result is a proposed new artifact.

## Tasks
- [x] Connect with Accessibility Ops on identified problems and proposed solutions
   - See [discussion thread on #accessibility-ops](https://dsva.slack.com/archives/C023ZKDJJK1/p1664468503794349)
- [x] Review submitted artifacts from recent staging reviews
   - See Quick Stats added above in this ticket.
- [x] Create follow-up ticket for implementing the proposed solution.
   - See [Create new accessibility artifact #47915](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47915)

## Peer Review
- [x] Ticket reassigned to peer reviewer when ready for peer review
- [x] Peer Reviewer: Completes review and validates user story/Acceptance Criteria are met

## Acceptance Criteria
- [x] Governance team has a solution for improving the communication between VFS and Gov on completed foundational accessibility testing
- [x] Governance team has a solution for connecting the known a11y issues to our Staging Review findings (aka documenting them in the DraftingTemplate)

## How to prepare this issue
- [x] Attach story to appropriate Initiative or Quarterly Tech-Debt Epic.
- [x] Add User Story, description, tasks, and Acceptance Criteria
- [x] Add any [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization).
- [x] Once all above items are checked, add the 'Ready' label.
- [x] Ticket has been discussed as a team at Planning.
- [x] Ticket has been sized as a team.
- [x] Ticket has been assigned to a Sprint.
- [x] Once assigned to the upcoming or current Sprint, move to the Planned pipeline.
- [x] Once assigned to the upcoming or current Sprint, add assignee and specify Peer Reviewer.

**Comment by briandeconinck** [comment]
@allison0034 This is ready for peer review. The output of this ticket is a proposed solution documented in a new issue, #47915. Let me know if you have any questions or anything doesn't make sense!

**Comment by allison0034** [comment]
@briandeconinck looks good, you can close
